renovate.json

```diff
@@ -110,6 +110,11 @@
     {
       "groupName": "@definitelytyped tools",
       "matchPackagePatterns": ["@definitelytyped/*"]
+    },
+    {
+      "groupName": "Lerna",
+      "matchPackageNames": ["lerna"],
+      "enabled": false
```

Comment:
This ideally has a `"description"` property with a short explanation on why it's disabled and a link to a github issue on the lerna repo.

Reply:
Created a PR for doing so: https://github.com/mui/material-ui/pull/40029 😊